### PR TITLE
Add image upload service and add images to seeded categories

### PR DESCRIPTION
**Pull request by LjSamardzic**
I hard coded one and the same image for all categories that get seeded at the outset, later on we should change that but just to have it for now. 

We send a relative url to the frontend and images can be accessed at baseUrl/relativeUrl stored in categories (uploads/categories/categoryId/timestamp_imageOriginalName)

### CODE REVIEW

File: src/main/kotlin/vegait/rs/osipodgorica/configuration/ResourceWebConfiguration.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+
+@Configuration
+internal class ResourceWebConfiguration : WebMvcConfigurer {
+
+    @Value("\${app.file.resource-location:file:uploads/}")
```
Comment (d-vucetic) [inline src/main/kotlin/vegait/rs/osipodgorica/configuration/ResourceWebConfiguration.kt:11]: Kakva je ovo sintaksa sa ovim  slashom \?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#### Windows
+./gradlew.bat build <br>
+./gradlew.bat bootRun
+
+Korisne informacije o gradle wrapper-u: 
+https://docs.gradle.org/current/userguide/gradle_wrapper.html
+https://www.youtube.com/watch?v=gKPMKRnnbXU&ab_channel=MarcoCodes
+
+### Ranovanje testova 
+(koje trenutno nemamo xD)
```
Comment (d-vucetic) [inline README.md:35]: Troll !

File: src/main/kotlin/vegait/rs/osipodgorica/service/ImageUploadService.kt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import java.nio.file.StandardCopyOption
+
+@Service
+class ImageUploadService(
+    @Value("\${app.file.upload-dir:./uploads}")
+    var uploadDir: String
+) {
+    val rootLocation: Path = Paths.get(uploadDir)
+
+    fun store(file: MultipartFile, folderName: String): String {
```
Comment (d-vucetic) [inline src/main/kotlin/vegait/rs/osipodgorica/service/ImageUploadService.kt:20]: Za ovo nisam siguran da li ces da dobijas multipart ili blob treba provjeriti sa masjom. Prije su imali ogranicenja sto se tice toga SPA nisam siguran sta je best practice

File: src/main/kotlin/vegait/rs/osipodgorica/configuration/ResourceWebConfiguration.kt (hunk 1 above)
Comment (LjSamardzic) [reply src/main/kotlin/vegait/rs/osipodgorica/configuration/ResourceWebConfiguration.kt:11]: Kotlin

**Review by d-vucetic — APPROVED** [review]
Mozemo multipart ostavit za sada da koristimo ali svakako dodati jos jedan endpoint ako bude potrebe

File: src/main/kotlin/vegait/rs/osipodgorica/configuration/ResourceWebConfiguration.kt (hunk 1 above)
Comment (LjSamardzic) [reply src/main/kotlin/vegait/rs/osipodgorica/configuration/ResourceWebConfiguration.kt:11]: https://stackoverflow.com/questions/46238790/how-to-use-spring-annotations-like-autowired-or-value-in-kotlin-for-primitive

File: README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+## Projekat za Udruženje paraplegičara Podgorice - Code For Cause 2023
+
+### Kako ranovati aplikaciju:
+1. Potrebno je skinuti JDK 17 i setovati JAVA_HOME env varijablu
```
Comment (d-vucetic) [inline README.md]: Da im mozda damo [https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.htm]( https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)

Comment (LjSamardzic) [reply README.md]: prije temurin jer se oracle JDK plaća u produkciji https://whichjdk.com/

https://adoptium.net/en-GB/temurin/releases/?version=17

**Review by d-vucetic — APPROVED** [review]

**Pull request merged**